cocotb/handle.py

```diff
@@ -530,11 +519,69 @@ def _range_iter(self, left, right):
 
     @NonHierarchyObject.value.getter
     def value(self):
-        # need to iterate over the sub-object
-        result = []
-        for x in range(len(self)):
-            result.append(self[x].value)
-        return result
+        """Get value of array by iterating through sub-handles in left-to-right order.
```

Comment:
This docstring should be phrased for the getter and setter combined. Replacing 'Get' with 'The', and 'the returned' with 'this' should resolve that.